README.rst

```diff
@@ -178,6 +181,13 @@ Command-line options
                         stored.
    --no-print-stream     disable print streaming
    --print-stream        Print tweet stream to stdout
+   --output-options      Set output format: 
+                         'r' Unmodified API Responses. (default).
+                         'a' Atomic Tweets: Tweet objects with expansions inline.
+                         'm' Message Stream: Tweets, Expansions, and Metadata
+                         as a stream of messages.
+   --atomic              Output "Atomic" Tweet format. 
+                         Equivalent to setting --output-format 'a'.
```

Comment:
Here `--output-format 'a'` vs`--output-options 'a'` which it actually is.

Reply:
ah good point, thanks - i'll stick with `--output-format`